digits/static/store.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <link rel="icon" href="/static/images/nvidia.ico"/>
+    <meta charset="UTF-8">
+    <title>Model Store</title>
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
Please don't add links that require internet access to download. Either check them into our source tree under `digits/static/js/` or pick another library.